webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
The reason I suggested that this be an overload is that I don't think people will have a convergent code path that covers both "I want everything" and "I want this specific thing".  The data that is returned and the use cases for it are quite different.  In that case you don't have a weird selector thing, you just have:

```js
function getAllStats(pc) { return pc.getStats(); }
function getTrackStats(pc, track) { return pc.getStats(track); }
```

BTW, just checked what we (gecko) do with extra arguments to functions when there are overloads or optional arguments.  Using a constructor that optionally initializes a value that defaults to `null`:

```idl
[Constructor(optional DOMString x)]
interface Foo { readonly attribute DOMString? x; };
```

Then you get this:
```
>>> new Foo()
Foo { x: null }
>>> new Foo('123')
Foo { x: "123" }
>>> new Foo(null)
Foo { x: null }
>>> new Foo(123)
TypeError: Argument 1 of Foo.constructor can't be converted to a string.
```

So there is no need to play games with types, pass the `null` if that is what you have.